draft-twi-reference-architecture.md

```diff
@@ -217,7 +217,10 @@ Executing the Workload inside a Trusted Execution Environment can lower the risk
 
 # Security Considerations
 
-<cref>TODO</cref>
+Maintaining security guarantees of Confidential Computing within the WIMSE reference architecture calls for adding the extensions specified in this draft.
```

Comment:
Here we should reference a Confidential Computing Document, which takes all the Security of the CC solutions into consideration that CC solutions require XYZ security aspects,  and then refer it here to mention they all apply to TWI based architecture...